content/concepts/kata/beta-process.md

```diff
@@ -5,98 +5,46 @@ sidebar: explanation-kata
+
+When kata is successfully solved, user unlocks further steps of the review process.
+
+### Satisfaction vote
+
+Users with [sufficient privileges](/references/gamification/privileges/#privileges) are able to vote on kata quality. They can cast one of three possible votes:
+
+- **Satisfied**
+- **Somewhat satisfied**
+- **Not satisfied**
+
+If you think the kata itself is valuable but you simply have some issues with it, then you should select "Somewhat" and create an issue instead. If you thought the kata is really great, other than the issues, then you can choose "Very". Please do not choose "None" simply because the kata has an issue but it is otherwise a good kata. The satisfaction rating is meant to indicate quality of kata as a concept, not its current state of condition in terms of open issues which can be fixed.
+
+### Difficulty rating vote
+
+Kata author assigns initial difficulty rank estimation to their kata, but they do not decide on its final rank. Difficulty rank is estimated from votes cast by users who solved the kata while it's in beta phase. This means that every user who solved a beta kata can give their own opinion on its difficulty, helping in determining its final rank.
+
+### Reviewing Code
+
+Users with sufficient priviledges can enter kata editor and view the code of all code snippets. Reading through them is a great way to find potential issues with tests, examples, or reference solution. Reviewers can check insides of the kata and verify whether it stands up to [Kata authoring best practices](/recipes/create-kata/best-practices/). Any discovered problems can be fixed right away in the code, or reviewer can raise issues or suggest improvements in kata discourse.
+
+### Reviewing Issues, Suggestions and Questions
+
+Some beta kata, especially ones created long time ago, can have some issues or suggestions which are still pending, but have been already fixed or are not applicable anymore because kata has changed. It's very helpful when such old issues are reviewed and marked as resolved if they are not valid anymore. It can also happen that there are posts in the kata discourse which report some problem, but are not labelled appropriately. It's generally a good idea to re-raise such issues as new posts with correct label, as it should help other users to find them and eventually fix them.
+
+## Kata Approval and Retirement
+
+After a kata has received [enough approval votes](/references/kata-ranking/approval-retire-criteria/) from the community, all issues are fixed, and community reached a consensus on its ranking, it's ready for the final step to come out of beta. At this point, sufficiently privileged user can review it to make sure all these criteria has been met (see [How to Review a Beta Kata](/recipes/review-beta/)), and release it as an official kata onto the site!
```

Comment:
- "and community reached ..." --> "and the community reached ..."
- "sufficiently privileged user" --> "sufficiently privileged users"
- "all these criteria has been met" --> "all these criteria have been met"

Reply:
```suggestion
After a kata has received [enough approval votes](/references/kata-ranking/approval-retire-criteria/) from the community, all issues are fixed, and the community reached a consensus on its ranking, it's ready for the final step to come out of beta. At this point, sufficiently privileged users can review it to make sure all these criteria have been met (see [How to Review a Beta Kata](/recipes/review-beta/)), and release it as an official kata onto the site!
```